### DISCUSSION TITLE
Handle 403 (Forbidden)

### DISCUSSION THREAD

**Discussion by tarakc02** [biglocalnews/clean-scraper#78] [Ideas]
See [this case from San Diego PD](https://www.sandiego.gov/police/data-transparency/mandated-disclosures/case?id=02-23-2000%202400%20Madison%20Avenue&cat=Officer%20Involved%20Shootings) for an example of a file that has metadata and looks scrape-able, but the "Redacted Documents" link returns a 403 status code. Would be nice to continue with other assets while also being able to keep track and report back which files are listed on an agency's site but not actually accessible.

**Comment by newsroomdev** [comment]
Interesting...this is less of an issue and more of a discussion. Converting this to a discussion thread to allow more space to talk about the where and how details of we would keep track of 403s

**Reply by newsroomdev** [reply]
Update @tarakc02: I saw this exact file in a recent Prefect run's log. Currently, we're logging these errors in Prefect, but it's not affecting the rest of the assets. Let me know if this covers what you had in mind and I can close this discussion
